integrations/systemd/m3dbnode.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description="M3DB Timeseries Database"
+Documentation=http://m3db.github.io/m3/
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/m3dbnode
```

Comment:
Curious if we should include the config flag within this ?

Reply:
Ah damn good catch, opened #974